From bd271425dc0c33b07109d348f1e4e3925674493d Mon Sep 17 00:00:00 2001
Message-ID: <bd271425dc0c33b07109d348f1e4e3925674493d.1707646257.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1707646257.git.sharlatanus@gmail.com>
References: <cover.1707646257.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 10 Feb 2024 17:23:32 +0000
Subject: [PATCH 20/30] gnu: go-golang-org-x-sync: Move to (gnu
 packages-build).

* gnu/packages/golang.scm (go-golang-org-x-sync): Move from here ...
* gnu/packages/golang-build.scm: ... to here.

Change-Id: I28b72dc45116398512c0255ac9e06b9c539e9e7b
---
 gnu/packages/golang-build.scm | 30 ++++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 30 ------------------------------
 2 files changed, 30 insertions(+), 30 deletions(-)

diff --git a/gnu/packages/golang-build.scm b/gnu/packages/golang-build.scm
index 8e64571929..7b8653960d 100644
--- a/gnu/packages/golang-build.scm
+++ b/gnu/packages/golang-build.scm
@@ -1,5 +1,6 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2020 Danny Milosavljevic <dannym@scratchpost.org>
+;;; Copyright © 2020 HiPhish <hiphish@posteo.de>
 ;;; Copyright © 2020 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
 ;;; Copyright © 2020 Vagrant Cascadian <vagrant@debian.org>
@@ -168,6 +169,35 @@ (define-public go-golang-org-x-net-html
     (description
      "This package provides an HTML5-compliant tokenizer and parser.")))
 
+(define-public go-golang-org-x-sync
+  (let ((commit "8fcdb60fdcc0539c5e357b2308249e4e752147f1")
+        (revision "1"))
+    (package
+      (name "go-golang-org-x-sync")
+      (version (git-version "0.1.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://go.googlesource.com/sync")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "07qrhni6f5hh5p95k1yk6s4wsj341q663irvx6rllrxfsymj6a0z"))))
+      (build-system go-build-system)
+      (arguments
+       `(#:import-path "golang.org/x/sync"
+         #:tests? #f
+         ;; Source-only package
+         #:phases (modify-phases %standard-phases
+                    (delete 'build))))
+      (home-page "https://go.googlesource.com/sync/")
+      (synopsis "Additional Go concurrency primitives")
+      (description "This package provides Go concurrency primitives in
+addition to the ones provided by the language and “sync” and “sync/atomic”
+packages.")
+      (license license:bsd-3))))
+
 (define-public go-golang.org-x-sync-errgroup
   (let ((commit "cd5d95a43a6e21273425c7ae415d3df9ea832eeb")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 7e5626a303..8bfabb9da9 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3559,36 +3559,6 @@ (define-public go-golang-org-x-image
 processing.")
       (license license:bsd-3))))
 
-(define-public go-golang-org-x-sync
-  (let ((commit "8fcdb60fdcc0539c5e357b2308249e4e752147f1")
-        (revision "1"))
-    (package
-      (name "go-golang-org-x-sync")
-      (version (git-version "0.1.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://go.googlesource.com/sync")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "07qrhni6f5hh5p95k1yk6s4wsj341q663irvx6rllrxfsymj6a0z"))))
-      (build-system go-build-system)
-      (arguments
-       `(#:import-path "golang.org/x/sync"
-         #:tests? #f
-         ;; Source-only package
-         #:phases (modify-phases %standard-phases
-                    (delete 'build))))
-      (synopsis "Additional Go concurrency primitives")
-      (description
-       "This package provides Go concurrency primitives in addition
-to the ones provided by the language and “sync” and “sync/atomic”
-packages.")
-      (home-page "https://go.googlesource.com/sync/")
-      (license license:bsd-3))))
-
 (define-public go-golang-org-x-text
   (package
     (name "go-golang-org-x-text")
-- 
2.41.0

